### PR TITLE
Silence unneeded exiv2/tiff debug output by default

### PR DESCRIPTION
**Pull request by pedrocr**
Our console output is quite noisy and exiv2 is a big culprit. I move the exiv2 and one tiff message to a new DT_DEBUG_CAMERA_SUPPORT that is disabled by default. This makes the output much nicer (from 5684 lines to 743 when exporting my full test collection).

### CODE REVIEW

**Comment by pedrocr** [comment]
I've also fixed a bug in colorin where it would try to fetch an embedded color profile from a .TIF file even if that file wasn't actually a TIFF but a RAW instead.

**Comment by pedrocr** [comment]
Merged manually

**Comment by LebedevRI** [comment]
The night (4 hours) from saturday to sunday can not be expected as a sufficient window to let others view the changes and leave comments.
Did this actually get reviewed by someone?

File: src/common/exif.cc [hunk 1]
```diff
@@ -2347,13 +2347,16 @@ int dt_exif_xmp_write(const int imgid, const char *filename)
 
 static void dt_exif_log_handler(int log_level, const char *message)
 {
-  if(log_level >= Exiv2::LogMsg::level()) fprintf(stderr, "[exiv2] %s\n", message);
+  if(log_level >= Exiv2::LogMsg::level())
+  {
+    // We don't seem to need \n in the format string as exiv2 includes it
+    // in the messages themselves
+    dt_print(DT_DEBUG_CAMERA_SUPPORT, "[exiv2] %s", message);
```
Comment (houz) [inline src/common/exif.cc]: missing `\n`.

Comment (pedrocr) [inline src/common/exif.cc]: No it's not, there's a comment explaining why just above the line

**Comment by pedrocr** [comment]
@LebedevRI It was discussed on IRC with hanatos and houz, since no real objections came up I just merged. I didn't even think this was PR-worthy but did it anyway.

**Pull request closed without merging**